core/common/src/ClipStyle.ts

```diff
@@ -97,6 +98,62 @@ export class CutStyle {
   }
 }
 
+/** Wire format describing a [[ClipHighlight]].
+ * @see [[ClipStyleProps.ClipHighlight]].
+ * @public
+ * @extensions
+ */
+export interface ClipHighlightProps {
+  /** Color to highlight the edge */
+  color?: RgbColorProps; // default white
+  /** Width of highlight in pixels */
+  width?: number; // in pixels, default 1
+}
+
+/** As part of a [[ClipStyle]], describes if/how edges of cut geometry should be highlighted.
+ * @note Edges are highlighted only if [[ClipStyle.ClipHighlight]] is `true`.
+ * @public
+ */
+export class ClipHighlight {
+  /** Color to highlight the edge, default to white */
+  public color?: RgbColor;
```

Comment:
Make all properties `readonly`. Otherwise people can do things like `viewport.displayStyle.clipStyle.clipHighlight.width = 5` and be confused when their view doesn't update.

Reply:
Please leave it to the original commenter to resolve their comments.

Reply:
sorry!